tests/test_concurrency.py

```
@@ -64,4 +64,121 @@ def test_record_concurrent_with_writes():
+    )
+
+
+def _create_index(r):
+    # First call creates the index; later calls error ("already indexed") but
+    # still plan as DDL and run through the GIL->L1-write DDL branch (eager
+    # create_rs_index / GilGuard). We only care that the command returns rather
+    # than hanging, so swallow the expected error.
+    try:
+        r.execute_command("GRAPH.QUERY", "test", "CREATE INDEX FOR (n:Node) ON (n.id)")
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)